packages/gatsby/src/bootstrap/load-config/index.ts

```diff
@@ -22,7 +18,6 @@ export async function loadConfigAndPlugins({
   program?: IProgram
 }): Promise<{
   config: any
```

Comment:
Can we type this?

Reply:
I opted to avoid making any extra changes to keep the scope smaller, but I can do a pass of these modules to update any existing missing types since there aren't too many cases